app/src/pages/trace/SpanAside.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { PropsWithChildren, useMemo } from "react";
+import { graphql, useRefetchableFragment } from "react-relay";
+import { css } from "@emotion/react";
+
+import { Flex, Text, View } from "@arizeai/components";
+
+import { AnnotationLabel } from "@phoenix/components/annotation";
+import { LatencyText } from "@phoenix/components/trace/LatencyText";
+import { SpanStatusCodeIcon } from "@phoenix/components/trace/SpanStatusCodeIcon";
+import { TokenCount } from "@phoenix/components/trace/TokenCount";
+import { useSpanStatusCodeColor } from "@phoenix/components/trace/useSpanStatusCodeColor";
+import { fullTimeFormatter } from "@phoenix/utils/timeFormatUtils";
+
+import { SpanAside_span$key } from "./__generated__/SpanAside_span.graphql";
+import { SpanAsideSpanQuery } from "./__generated__/SpanAsideSpanQuery.graphql";
+
+const annotationListCSS = css`
+  display: flex;
+  padding-top: var(--ac-global-dimension-size-50);
+  flex-direction: column;
+  gap: var(--ac-global-dimension-size-100);
+  align-items: flex-start;
+`;
+
+/**
+ * A component that shows the details of a span that is supplementary to the main span details
+ * @returns
+ */
+export function SpanAside(props: { span: SpanAside_span$key }) {
+  const [data] = useRefetchableFragment<SpanAsideSpanQuery, SpanAside_span$key>(
+    graphql`
+      fragment SpanAside_span on Span
+      @refetchable(queryName: "SpanAsideSpanQuery") {
+        id
+        code: statusCode
+        startTime
+        endTime
+        tokenCountTotal
+        tokenCountPrompt
+        tokenCountCompletion
+        spanAnnotations {
+          id
+          name
+          label
+          annotatorKind
+          score
+        }
+      }
+    `,
+    props.span
+  );
+  const {
+    startTime,
+    endTime,
+    code,
+    tokenCountCompletion,
+    tokenCountPrompt,
+    tokenCountTotal,
+  } = data;
+  const startDate = useMemo(() => new Date(startTime), [startTime]);
+  const endDate = useMemo(
+    () => (endTime ? new Date(endTime) : null),
+    [endTime]
+  );
```

Comment:
is it necessary to memoize these values? seems like a pretty cheap operation

Reply:
I don't think constructing a date object is no cost so I'm gonna leave in.

Reply:
gotcha, recently this has come up over on the arize side so was curious here, the cost of memos might be higher than the cost of low cost operations like this